Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/BreadCrumbs.cpp

```diff
@@ -98,6 +99,7 @@ namespace AzQtComponents
         m_lineEdit->installEventFilter(this);
         connect(m_lineEdit, &QLineEdit::returnPressed, this, &BreadCrumbs::confirmEdit);
         Style::flagToIgnore(m_lineEdit);
+        m_lineEdit->setStyleSheet("QLineEdit {color: black; background-color: white;}");
```

Comment:
Should do this in the breadcrumbs qss file instead:

`Code\Framework\AzQtComponents\AzQtComponents\Components\Widgets\BreadCrumbs.qss`

It already has a heading for `AzQtComponents--BreadCrumbs QLineEdit`, and actually has some conflicting properties set in there (e.g. color: white;) so should look into if there are other QLineEdit's in the breadcrumb widget you might need to account for.

Reply:
Oh cool! Didn't know this file existed, will fix it there Thanks.